src/library_fs.js

```diff
@@ -165,6 +166,12 @@ FS.staticInit();
       get isDevice() {
         return FS.isChrdev(this.mode);
       }
+      get timestamp() {
+        return this.atime;
+      }
+      set timestamp(val) {
+        this.atime = this.mtime = this.ctime = val;
+      }
```

Comment:
What still uses `.timestamp`?

Reply:
Well I didn't change `library_idbfs` because I didn't understand what was going on, so I put this in to allow it to keep working as-is. But because we have this setter there are several places in the new code where instead of writing
```js
node.atime = node.mtime = node.ctime = val
```
I wrote
```js
node.timestamp = val
```

Reply:
I believe `library_idbfs.js` has its own `timestamp` field on its own objects, but it doesn't use that property name on FS objects. So I think we can remove this getter/setter pair. If I am wrong we should see the IDBFS tests error.